ops/lib/__init__.py

```diff
@@ -185,7 +185,7 @@ def _parse_lib(spec):
     logger.debug("    Parsing %r", spec.name)
 
     try:
-        with open(spec.origin, 'rt', encoding='utf-8') as f:
+        with open(spec.origin, encoding='utf-8') as f:
```

Comment:
This feels like something that might be redundant with a default, but I don't mind being explicit to the developers as to expected behavior. Do you know why this was removed?

Reply:
Yeah, I guess pyupgrade removes it because it's the default (now? not sure how long that's been). Read/r has been the default forever, maybe the text/t default came later. But I can see where this is debatable, so I've reverted this change and just left it explicit.